unity/Assets/Scripts/Content/DictionaryI18n.cs

```diff
@@ -262,7 +262,7 @@ public void flushRaw()
             {
                 string key = rawLine.Split(COMMA)[0];
                 // Process non repeated list line
-                if (!dict.ContainsKey(key))
+                if (key != "." && !dict.ContainsKey(key))
```

Comment:
This fix is fine, but I would prefer that separate issues get their own pull request.